### ISSUE TITLE
Document default JVM DNS caching behavior and how to configure it

### ISSUE THREAD

**Issue by jakommo** [elastic/logstash#11731]
Similar to [Elasticsearch, we should document default JVM DNS cache settings](https://www.elastic.co/guide/en/elasticsearch/reference/current/networkaddress-cache-ttl.html) and how to configure it.
Currently we only document it on the [Elasticsearch output](https://www.elastic.co/guide/en/logstash/current/plugins-outputs-elasticsearch.html#_dns_caching), but since this applies to many (all?) outputs that use a DNS name, we should document this globally for Logstash. 

cc @jsvd

**Comment by lhirlimann** [comment]
@jakommo would a KB help with this?

**Comment by jakommo** [comment]
A KB is better than nothing, but I think it would best fit into the product documentation

**Comment by lhirlimann** [comment]
> A KB is better than nothing, but I think it would best fit into the product documentation

Agreed. Any how I made a draft KB https://support.elastic.dev/knowledge/view/7716cc73
